### ISSUE TITLE
Type checking on `<nuxt-link>` not working when import from `#components` is present

### ISSUE THREAD

**Issue by andreww2012** [victorgarciaesgi/nuxt-typed-router#109]
**Describe the bug**
Type checking of `<nuxt-link>`'s `to` prop stops working if an import statement from `#components` is present in SFC, but only after running `nuxt dev`.

**Expected behavior**
Errors are not ignored.

**Environnement infos**

```
Nuxi 3.6.5                                                                                                                                                                                                                                                                                            
- Operating System: Windows_NT
- Node Version:     v16.16.0
- Nuxt Version:     3.6.5
- Nitro Version:    2.5.2
- Package Manager:  pnpm@8.6.10
- Builder:          vite
- User Config:      devtools, modules, imports
- Runtime Modules:  nuxt-typed-router@3.2.5
- Build Modules:    -
```

**Your `pages` folder structure**
**Your nuxt.config.ts**

Reproduction with instructions: https://github.com/andreww2012/nuxt-typed-router-no-errors-when-adding-components-import

**Comment by victorgarciaesgi** [comment]
Fixed in `3.3.1` @andreww2012 , keep me updated!

**Comment by andreww2012** [comment]
Thank you, it's working now! 🥇
